components/basecomponents/Exercise/components/ActionButton.tsx

```diff
@@ -50,14 +54,16 @@ export const ActionButton = forwardRef(
         ref={btnRef}
         buttonStyle="primary"
         onClick={async (e) => {
+          isPlausible && plausible('StartedExerciseEvent', { props: { length_of_exercise: exerciseLength } });
```

Comment:
tady může dojít k tomu že dvojklik se zaregistruje jako dvě události, mělo by se to volat níže až po nastavení obou pending na true